tools/mdanalysis/ramachandran_auto_protein.xml

```diff
@@ -0,0 +1,143 @@
+<tool id="mdanalysis_ramachandran_protein" name="Ramachandran Analysis" version="@VERSION@+galaxy@GALAXY_VERSION@">
+  <description>- Ramachandran plot for proteins</description>
+  <macros>
+    <import>macros.xml</import>
+    <token name="@GALAXY_VERSION@">0</token>
+  </macros>
+  <expand macro="requirements">
+    <requirement type="package" version="0.9.0">seaborn</requirement>
+    <requirement type="package" version="2.10.0">h5py</requirement>
+    <requirement type="package" version="2.11.1">jinja2</requirement>
+  </expand>
+  <command detect_errors="exit_code">
+    <![CDATA[
+        python '$__tool_directory__/ramachandran_auto_protein.py'
+            --itraj '$trajin'
+            --istr '$strin'
+            --itrajext '$trajin.ext'
+            --istrext '$strin.ext'
+            --isegid1 '$segid1'
+            #if $subselection.residues == 'Yes':
+                --iresid1 '$subselection.resid1'
+                --iresid2 '$subselection.resid2'
+            #end if
+            #if $subselectionr.residuenames == 'Yes':
+                --iresname '$subselectionr.resname'
+            #end if
+            --itemplatepath '$__tool_directory__'
+            #if $groupby:
+                --igroupby '$groupby'
+            #end if
+            --o_plot1 '$o_plot1'
+            --o_plot2 '$o_plot2'
+            --o_data1 '$o_data1'
+            --o_html1 '$o_html1'
+        2>&1
+]]>
+  </command>
+  <inputs>
+    <expand macro="analysis_inputs"/>
+    <param name="segid1" type="text" value="PROA" label="Segment ID of atom 1"/>
```

Comment:
this needs a validator, I think we have in other tools already some which you can reuse.